pom.xml

```diff
@@ -325,6 +325,17 @@
     </extraJavaTestArgs>
   </properties>
   <repositories>
+    <repository>
```

Comment:
Why is this required?

Reply:
man need this to download parquet 1.12.4.